qcodes/math/field_vector.py

```diff
@@ -1,11 +1,3 @@
-"""
```

Comment:
Note that we strongly encourage the use of module docstrings. I agree that this particular docstring should be moved to the class, but it's worth thinking about putting something else in here. It can be as brief as the one here: https://github.com/QCoDeS/Qcodes/blob/master/qcodes/utils/types.py